Bottom: 357f58cb80f02809d85d51d1bdddfd9b08caab95
Top:    6882730bf5edd87a67e5bdb10ff7563857430a15
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:14 +0000

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
